tests/cypress/e2e/unit_tests/p0_fleet.spec.ts

```diff
@@ -16,7 +16,7 @@ import 'cypress/support/commands';
 import { qase } from 'cypress-qase-reporter/dist/mocha';
 
 export const appName = "nginx-keep";
-export const clusterName = "k3d-imported";
+export const clusterName = "imported-0";
```

Comment:
Just sharing a thought:

If I am not mistaken with the current proposal all 3 ds clusters would contained the pushed app. Being that case I wonder if this clusterName should be kept here or :
- scoped locally depending on the clusterName where the app is deployed.
- add the other 2 imported clusters to check in all.

Reply:
Right, this const is used only by call `cy.checkApplicationStatus(appName, clusterName);` so I guess it would be nice to check app presence on all targeted clusters instead.